### ISSUE TITLE
Bug: Autostart Playback option does not auto start playback

### ISSUE THREAD

**Issue by Gero5** [mpogue2/SquareDesk#800]
menu: Music/Autostart Playback, click on the line. 
Autostart Playback has now a checked mark (active) in the menu
double klick on singing audio in the list - it does not start
double klick on patter audio - it does not start

**Comment by mpogue2** [comment]
Thanks for the report!

Fixed by: 6cca55c7222404f2cba8dd8cb95b581681aabdd7

Autostart stopped working a while ago, most likely when we switched to asynchronous MP3 loading (when we moved away from the non-open-source libbass library). In the new asynchronous design, when the Play button was auto-pressed at double-click time, the song load was started, but it actually wasn't done loading, and so the play request was ignored.  Moved autostart detection to the second (final) stage of loading, and it appears to work as-designed now.
